packages/@okta/vuepress-site/docs/reference/api/threat-insight/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: ThreatInsight
+category: management
+---
+
+# ThreatInsight Configuration API
+
+[Okta ThreatInsight](https://help.okta.com/en/prod/Content/Topics/Security/security-threat-insight.htm) maintains a constantly evolving list of IPs that exhibit suspicious behaviors suggestive of malicious activity. Authentication requests associated with an IP in this list can be logged in [System Log](https://help.okta.com/en/prod/Content/Topics/Reports/Reports_SysLog.htm?cshid=ext_Reports_SysLog) and blocked. The Okta ThreatInsight Configuration API provides operations to manage your ThreatInsight configuration.
+
+
+## ThreatInsight Configuration object
+
+| Field Name     | Description                                                         	| Data Type                                     | Required      | Max Length    |
+| :------------- | :------------------------------------------------------------------	| :-------------------------------------------- | :------------ | :------------ |
+| action         | Specifies how Okta would respond to authentication requests from suspicious IPs. Value is one of `none`, `audit`, `block`. With `none`, ThreatInsight will be disabled; with `audit`, Okta will log the suspicious requests in System Log; with `block`, Okta will log the suspicious requests in System Log and block the requests. | String                                        | Yes		| N/A           |
+| excludeZones   | Accepts a list of [Network Zone](/docs/reference/api/zones/) IDs. IPs in the excluded Network Zones will not be logged or blocked by Okta ThreatInsight and will proceed to Sign On rules evaluation. This will ensure that traffic from known, trusted IPs is not accidentally logged or blocked.	| List	                                        | No		| N/A           |
+
+
+## ThreatInsight Configuration API Operations
+
+### Update ThreatInsight configuration
+
+<ApiOperation method="post" url="/api/v1/threats/configuration" />
+
+Update ThreatInsight configuration
+
+#### Request Example
+
+```bash
+curl -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+    "action": "audit",
+    "excludeZones": ["nzo1q7jEOsoCnoKcj0g4"],
+}' "https://${yourOktaDomain}/api/v1/threats/configuration"
+```
+
+#### Successful Response Example
```

Comment:
Are you planning to provide an unsuccessful response example? If not, then this can just be: Response example